src/sql/azdata.proposed.d.ts

```diff
@@ -403,6 +403,25 @@ declare module 'azdata' {
 		title: string;
 	}
 
+	export interface ConnectionProvider extends DataProvider {
+		/**
+		 * Changes the password for a connection profile (in case of the password expiration scenario)
```

Comment:
I don't think we need to specify that this is for password expiration. After all -there's nothing stopping someone from calling changePassword directly on a provider (by calling azdata.dataprotocol.getProvider)

```suggestion
		 * Changes the password for a connection profile
```